contracts/mocks/AirdropCampaignDataUpgrade.sol

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/// @title AirdropCampaignData - Data contract for storing of daily hashes of airbro Campaigns
+contract AirdropCampaignDataUpgrade is Initializable, UUPSUpgradeable, OwnableUpgradeable {
```

Comment:
This contract can inherit the original one, instead of copy / pasting the content, and have that added function in it